packages/base/src/Base.ts

```diff
@@ -163,3 +163,47 @@ export function SpectrumMixin<T extends Constructor<ReactiveElement>>(
 }
 
 export class SpectrumElement extends SpectrumMixin(LitElement) {}
+
+if (window.__swc.DEBUG) {
+    window.__swc = {
+        ...window.__swc,
+        issuedWarnings: new Set(),
+        issueWarning: (element, data, message, url, issues): void => {
```

Comment:
Perhaps consider the verb form `warn()` ala `console`.

It would also be 👍 if we could figure out a way to reduce the number of arguments required here.

```suggestion
        issueWarning: (element, data: SWCWarningData, message: string, url: string, issues: string[]): void => {
```

Reply:
`.warn()` sets a nice tone for expanding to `.log()` and `.error()` along the lines of @castastrophe's note later down the line if we need.

We type this method in `projects/types/global.d.ts` does it need to be typed here, too? Can add for coverage, however.

Would an options bag be better for this method?

Reply:
In that case it probably doesn't need to be typed (I didn't check in an editor).

If we used an options bag with some reasonable defaults, could we flatten out the nested props in `SWCWarningData`? Such that both of these would be valid uses:

`.warn(this, "Don't do that", 'https://example.com')`

`.warn(this, 'Or that either', 'https://adobe.com', {
  level: 'high',
  type: 'accessibility',
});`

Alternatively, with tags:

`.warn(this, "Don't do that", 'https://example.com', ['high', 'accessibility']);`

Reply:
Updated with that. Good call on the defaults for simplicity.